chef/cookbooks/cpe_chrome/recipes/windows_chrome.rb

```diff
@@ -0,0 +1,48 @@
+#
+# Cookbook Name:: cpe_chrome
+# Recipe:: windows_chrome
+#
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+#
+# Copyright (c) 2016-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+
+chrome_installed =
+  File.file?("#{ENV['ProgramFiles(x86)']}\\Google\\Chrome\\" \
+             'Application\\chrome.exe')
+
+return unless chrome_installed
+
+# We've been seeing users running development VMs/hardware that do not have SID
+# because of no intern person data. Also freshly imaged machines do not have
+# intern person data until the laptop is used. So we will (for now) exit out of
+# the cookbook to accomodate that scenario.
+uid = node.dig('cpe/person/uid')
+return if uid.nil?
+
+reg_settings = []
+node['cpe_chrome'].each do |setting|
```

Comment:
not run time safe. but this whole thing might need to be reworked.... good for now